src/Glpi/Exception/Access/RequiresHttpsCookieOnlyException.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Glpi\Exception\Access;
+
+use Glpi\Application\View\TemplateRenderer;
+use Symfony\Component\HttpFoundation\Response;
+
+class RequiresHttpsCookieOnlyException extends AccessException
```

Comment:
It is not the secured cookie configuration that is required, but the secured HTTP context that is required (because `session.cookie_secure` is On on the server).


```suggestion
class RequiresHttpsException extends AccessException
```